src/glang/lib/std/io.c3

```diff
@@ -81,16 +102,6 @@ function consume_length : (io : BufferedFile mut&, length : isize) -> void = {
     io.buffer_pointer += length;
 }
 
-function read : (file : BufferedFile&, buffer: Span<u8>) -> View<u8> = {
-    return file.underlying_file:read(buffer);
-}
```

Comment:
I guess we don't need this

Reply:
Yeah + it was a bit misleading since it didn't actually buffer the read. :smile:

Reply:
As you'd expect ;)